src/SearchResults.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import Moment from "react-moment";
+import moment from "moment";
+
+const SearchResults = props => {
+  let isClicked = false;
+  // function theRowisclicked(){
+  //   if (!isClicked){
+  //     isClicked=true;
+  //     e.target.classList.add("highLight")
+
+  //   }
+  //   else{
+  //     isClicked=false;
+  //     e.target.classList.remove("highLight")
+  //   }
+  // }
+  return (
+    <div className="table-responsive-sm">
+      <table className="table table-bordered">
+        <thead className="table-primary fontColor">
+          <tr
+            onClick={e => {
```

Comment:
Table header should not have `onClick` event. It should be only applied to each table row inside table body.